src/Abp.MailKit/EmailExtensitons.cs

```diff
@@ -1,30 +1,269 @@
+                    break;
+                case MailPriority.Low:
+                    message.Headers.Replace(HeaderId.Priority, "non-urgent");
+                    message.Headers.Replace(HeaderId.Importance, "low");
+                    message.Headers.Replace(HeaderId.XPriority, "4 (Low)");
+                    break;
+            }
+
+            if (!string.IsNullOrEmpty(mail.Body))
+            {
+                var text = new TextPart(mail.IsBodyHtml ? "html" : "plain");
+                text.SetText(mail.BodyEncoding ?? Encoding.UTF8, mail.Body);
+                body = text;
+            }
+
+            if (mail.AlternateViews.Count > 0)
+            {
+                var alternative = new MultipartAlternative();
+
+                if (body != null)
+                {
+                    alternative.Add(body);
+                }
+
+                foreach (var view in mail.AlternateViews)
+                {
+                    var part = GetMimePart(view);
+
+                    if (view.BaseUri != null)
+                        part.ContentLocation = view.BaseUri;
+
+                    if (view.LinkedResources.Count > 0)
+                    {
+                        var type = part.ContentType.MediaType + "/" + part.ContentType.MediaSubtype;
+                        var related = new MultipartRelated();
+
+                        related.ContentType.Parameters.Add("type", type);
+
+                        if (view.BaseUri != null)
+                            related.ContentLocation = view.BaseUri;
+
+                        related.Add(part);
+
+                        foreach (var resource in view.LinkedResources)
+                        {
+                            part = GetMimePart(resource);
+
+                            if (resource.ContentLink != null)
+                                part.ContentLocation = resource.ContentLink;
+
+                            related.Add(part);
+                        }
+
+                        alternative.Add(related);
+                    }
+                    else
+                    {
+                        alternative.Add(part);
+                    }
+                }
+
+                body = alternative;
+            }
+
+            if (body == null)
+            {
+                body = new TextPart(mail.IsBodyHtml ? "html" : "plain");
+            }
+
+            if (mail.Attachments.Count > 0)
+            {
+                var mixed = new Multipart("mixed");
+
+                if (body != null)
+                {
+                    mixed.Add(body);
+                }
+
+                foreach (var attachment in mail.Attachments)
+                {
+                    mixed.Add(GetMimePart(attachment));
+                }
+
+                body = mixed;
+            }
+
+            message.Body = body;
 
             return message;
         }
+
+        static MimePart GetMimePart(AttachmentBase item)
```

Comment:
Make default `private` access modifier obvious (same for `ToMailboxAddress` and `ToInternetAddressList`)